### ISSUE TITLE
Where does the list of countries in ISO 3166-1 come from?

### ISSUE THREAD

**Issue by johnwcowan** [semanticarts/gist#120]
Tl;dr:  From the [UN M49 codes](https://en.wikipedia.org/wiki/UN_M49).

But what is UN M49?  A set of 3-digit codes assigned by the UN Statistics Division which uniquely identify the following:

1. Individual geopolitical entities.  These are coded separately if they are in some sense separate economies.  Thus Greenland is a political dependency of Denmark, but has its own code because it is not integrated into the Danish economy: for example, it's not part of the EU.  This is also why "Taiwan, Province of China" (as it is officially called in this context) has its own code.

   The ISO 3166-1 standard simply maps these country codes into 2-letter codes.   However, if a country substantially changes, its M49 code may change while its 2-letter code remains the same. 
 When West and East Germany merged, their numeric codes were retired, while the 2-letter country code of West Germany changed its meaning to the whole of Germany, which got a new numeric code. 

1. Regions containing two or more countries, known as the [U.N. geoscheme](https://en.wikipedia.org/wiki/United_Nations_geoscheme).  These are organized in a hierarchical scheme.  For example, 001 represents all the countries in the world, and 155 represents the countries in Western Europe, which for this purpose are precisely Austria, Belgium, France, Germany, Liechtenstein, Luxembourg, Monaco, the Netherlands, and Switzerland.    

   Parts of the world that are not part of any country are excluded: there is no code for the Pacific Ocean, though there is a code for Polynesia, a region of ten island nations in the Pacific.

3. Miscellaneous things.  There is a code for the Commonwealth of Independent States, an intergovernmental organization that contains some but not all of the countries that came into existence when the Soviet Union broke up.   Similarly, there are codes for all landlocked developing countries, all island developing countries, and all least developed countries.

Just sayin'.

**Comment by rjyounes** [comment]
Not a gist question.
